recipes-core/base-files/base-files/system-backup.txt

```diff
@@ -5,7 +5,15 @@
 /etc/group
 /etc/gshadow
 /etc/sudoers.d
+/etc/hostname
+/etc/hosts
 /etc/systemd/network/
+/etc/ssh/ssh_host_ecdsa_key
+/etc/ssh/ssh_host_ecdsa_key.pub
+/etc/ssh/ssh_host_ed25519_key
+/etc/ssh/ssh_host_ed25519_key.pub
+/etc/ssh/ssh_host_rsa_key
+/etc/ssh/ssh_host_rsa_key.pub
```

Comment:
Not backing up host keys was an intentional decision by @jklare IIRC.

Reply:
I can not recall the reasoning for not backing up host keys and would think that it might confuse users if they run an upgrade and afterword the host key is regenerated.

Reply:
I faintly remember the argument was about making the user aware that the system was upgraded/changed. Though I don't have particular feelings there.